redbeat/schedulers.py

```diff
@@ -455,6 +456,9 @@ def maybe_due(self, entry, **kwargs):
         return next_time_to_run
 
     def tick(self, min=min, **kwargs):
+        if self.lock_acquiring_error:
```

Comment:
This is the earliest place I believe we can successfully throw. We can't check and throw in somewhere like setup_schedule as that has already been called by the time `acquire_distributed_beat_lock` is called. After `acquire_distributed_beat_lock` this is the next redbeat function that is called by beat and hence the earliest place we can throw if `acquire_distributed_beat_lock` fails.